flink-autoscaler-standalone/README.md

```diff
@@ -0,0 +1,79 @@
+# Flink Autoscaler Standalone
+
+## What's the autoscaler standalone?
```

Comment:
We should add a paragraph clarifying that this should only be used in non-kubernetes environments. For Kubernetes the Operator provides a full integration that is preferable.